x-pack/plugins/ingest_manager/common/types/rest_spec/datasource.ts

```diff
@@ -16,6 +19,14 @@ export interface GetOneDatasourceRequest {
   };
 }
 
+export interface GetDatasourcesResponse {
```

Comment:
:) I just added this type to my PR as well - https://github.com/elastic/kibana/pull/63373

Reply:
@jen-huang is working on datasources table and probably adding it as well!

Reply:
@neptunian fwiw, I'm working on Datasources table in https://github.com/elastic/kibana/issues/57961